service/controller/app/v1/key/key.go

```diff
@@ -1,12 +1,26 @@
 package key
 
 import (
+	"fmt"
+
 	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
 	"github.com/giantswarm/microerror"
 
 	"github.com/giantswarm/app-operator/pkg/label"
 )
 
+// AppConfigMapName returns the name of the configmap that stores app level
+// config for the provided app CR.
```

Comment:
Previous `ConfigMapName` function wasn't clear. This gets the name of the configmap that stores app level config for this CR.